### PR TITLE
AG-13502 Polar series z-index order

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-13502

### CODE REVIEW

File: packages/ag-charts-community/src/chart/__image_snapshots__/integrated-charts-examples-test-ts-integrated-charts-examples-changing-chart-type-for-donut-basic-it-should-render-to-canvas-as-expected-1-snap.png
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/__image_snapshots__/integrated-charts-examples-test-ts-integrated-charts-examples-changing-chart-type-for-donut-basic-it-should-render-to-canvas-as-expected-1-snap.png:1]: Should the labels have disappeared here?

File: packages/ag-charts-enterprise/src/series/nightingale/nightingaleSeries.ts [hunk 1]
```diff
@@ -21,6 +21,16 @@ export class NightingaleSeries extends RadialColumnSeriesBase<_ModuleSupport.Sec
         super(moduleCtx, { animationResetFns: { item: resetNightingaleSelectionFn } });
     }
 
+    override setSeriesIndex(index: number) {
+        if (!super.setSeriesIndex(index)) return false;
+
+        // Ensures highlights always appear on top
+        this.contentGroup.zIndex = [0, PolarZIndexMap.FOREGROUND, index];
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/nightingale/nightingaleSeries.ts:28]: `this.labelGroup` too?

File: packages/ag-charts-community/src/chart/__image_snapshots__/integrated-charts-examples-test-ts-integrated-charts-examples-changing-chart-type-for-donut-basic-it-should-render-to-canvas-as-expected-1-snap.png
Comment (jacobp100) [reply packages/ag-charts-community/src/chart/__image_snapshots__/integrated-charts-examples-test-ts-integrated-charts-examples-changing-chart-type-for-donut-basic-it-should-render-to-canvas-as-expected-1-snap.png:1]: No. Good catch. I was wondering what the difference way

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**